Hazel/CMakeLists.txt

```diff
@@ -0,0 +1,103 @@
+cmake_minimum_required(VERSION 3.16)
+project(Hazel)
+
+set(CMAKE_CXX_STANDARD 17)
+
+add_compile_definitions(HZ_DEBUG)
```

Comment:
This adds this define for all build types right? Probably best to only include it in debug only with something like, add_compile_definiations($<$CONFIG:Debug:HZ_DEBUG>)